dags/hindawi/hindawi_api_client.py

```diff
@@ -29,7 +29,27 @@ def get_articles_metadata(self, parameters, doi=None):
                 break
             return response_content
 
-    # TODO: don't forget to download pdf when it will be needed
+    def get_file(self, parameters, extension, doi=None):
+        self.base_url = "http://downloads.hindawi.com"
+        self.path_segments = ["journals", "ahep"]
+
+        doi_part = doi.replace("10.1155/", "") + f".{extension}"
```

Comment:
what if `doi` is None? it will break. Also why hardcoded `10.1155`?

Reply:
https://github.com/SCOAP3/scoap3-next/blob/a9763de5886c64461ad901ad833941ada153e558/scoap3/modules/workflows/workflows/articles_upload.py#L302

Reply:
We have to understand why this is there, could you please ask Anne?

Reply:
Asked her, she will ask hindawi

Reply:
Anne told, me that this situation is for all articles.
The first part of the doi reflects the publisher, which we are cutting off.
It is the way how Hindawi api works